app/views/admin_users/rewards/show.html.erb

```diff
@@ -1,8 +1,16 @@
 <div class="shadow p-3 mb-5 bg-body rounded">
   <figure class="text-end">
     <figcaption class="block-footer">
-      <strong>Price:</strong>
-      <%= reward.price %>
+      <div class="row">
+      <div class="col-6 text-start">
+        <strong>Category:</strong>
+        <%= reward.category.present? ? reward.category.title : 'without category' %>
```

Comment:
We are using this check `reward.category.present?` in views a few times. I would suggest moving this to the reward model method that returns `category.title` or `'without category'`

Reply:
Thanks a lot! I changed it 👍